sample-queries/sample-queries.json

```diff
@@ -64,13 +64,13 @@
         "skipTest": false
     },
     {
-        "id": "6d9500f7-24b1-4f3f-a76f-20a4c8f89dcf",
+        "id": "58e876d9-2491-4335-a2e5-e60515c541ec",
         "category": "Getting Started",
         "method": "GET",
         "humanName": "my To Do task lists",
-        "requestUrl": "/beta/me/todo/lists",
+        "requestUrl": "/v1.0/me/todo/lists",
```

Comment:
Is this API ready because I get this error when I try it on Graph Explorer? 
![image](https://user-images.githubusercontent.com/4668452/97003510-30770d00-1544-11eb-8226-71b86ab7a1e9.png)

Reply:
The API changes have been deployed, you can retry it now